src/Stryker.Core/Stryker.Core.UnitTest/Mutants/TestResources/Mutator_FromActualProject_IN.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Test 
```

Comment:
From what actual project is this code and do we have correct attribution /licensing to use it?

Reply:
NFluent, comment added

Reply:
Cool! That should avoid any confusion anyone might have :)